ups/product_deps

```diff
@@ -37,8 +37,8 @@ table_fragment_end
 # Add the dependent product and version
 
 product                   version
-sbncode                   v09_42_04
-icarusalg                 v09_42_00
+sbncode                   v09_43_00
+icarusalg                 v09_43_00
```

Comment:
I am not sure if this is not reviewed/approved by others by tomorrow these versions will have to be updated to be merged next week. Version will change tomorrow (Friday)with weekly release